apps/prairielearn/src/pages/partials/navbarInstructor.ejs

```diff
@@ -70,18 +91,29 @@
             </li>
         <% } %>
     <% } else { %>
-        <% if ((authz_data.has_course_permission_preview || authz_data.has_course_instance_permission_view) && authz_data.course_instances && authz_data.course_instances.length > 0) { %>
-            <li class="navbar-text mx-2 no-select">/</li>
+        <li class="navbar-text mx-2 no-select">/</li>
 
-            <li class="nav-item dropdown">
-                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuInstanceChoose" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                    Choose course instance...
-                </a>
-                <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuInstanceChoose">
-                    <% authz_data.course_instances.forEach(function(ci) { %>
-                        <a class="dropdown-item <% if (ci.disabled) { %>disabled<% } %>" href="<%= plainUrlPrefix %>/course_instance/<%= ci.id %>/instructor/instance_admin<% if (navPage == "instance_admin" && typeof navSubPage !== 'undefined' && navSubPage != "file_edit") { %>/<%= navSubPage %><% } %>"><%= ci.short_name %></a>
```

Comment:
Two notes here:

- AFAICT `ci.disabled` simply isn't a thing, and so this should be safe to remove. does that sound correct?
- I'm not entirely sure what's going on with the construction of the URL here. It seems like it's trying to preserve a subpage when switching instances, but from what I can see, `navPage == "instance_admin"` will _never_ be true because this entire branch is only hit when we're specifically not on a course instance page. So, I opted to forgo this and just link directly to the course instance admin homepage.

Reply:
I agree that `ci.disabled` doesn't seem to exist (I can't wait for more TS!). I think it's fine to unconditionally link to the admin page.